tests/integration/test_upgrade.py

```diff
@@ -0,0 +1,52 @@
+import pytest
```

Comment:
I didn't see any existing integration tests for the `upgrade` command, so I created this file.
I am attempting to match the pattern of test format from other tests.
Let me know if you want me to put these somewhere else.